modAionImpl/src/org/aion/zero/impl/db/DBUtils.java

```diff
@@ -703,4 +706,117 @@ public AionBlockSummary getRight() {
 
         System.out.println("Importing stored blocks COMPLETE.");
     }
+
+    /**
+     * @implNote Used by the CLI call.
+     */
+    public static Status queryTransaction(byte[] txHash) {
```

Comment:
Please add some unit tests to cover these methods.

Reply:
Added the tests for covering the methods.